app/controllers/users_controller.rb

```diff
@@ -1,9 +1,14 @@
+require 'set'
+
 class UsersController < ApplicationController
   include SocialAccountsHelper
-  before_action :authorize
+  include UsersHelper
+  before_action except: %i[show index search]
```

Comment:
What does this do? I would expect the symbol of a method name here that would be executed before the specified actions

Reply:
I added `before_action :authorize` again.